composer.json

```diff
@@ -21,10 +21,10 @@
     ],
     "require": {
         "php": "^7.0",
-        "illuminate/support": "5.5.*"
+        "illuminate/support": "5.5.*|5.6.*"
     },
     "require-dev": {
-        "orchestra/testbench": "3.5.*",
```

Comment:
Shouldn't we use `3.5.*|3.6.*` here so the tests can use Laravel 5.5 when using lower deps?

Reply:
Good catch!